content-services/community/install/ansible.md

```diff
@@ -0,0 +1,390 @@
+
+* Share: `http://<target-host-ip>/share`
+* Repository: `http://<target-host-ip>/alfresco`
+* API Explorer: `http://<target-host-ip>/api-explorer`
+
+### Multi-machine installation
+
+The diagram below shows the result of a multi-machine installation.
+
+![acs-multi-machine]({% link content-services/images/acs-multi-machine.png %})
+
+Once you've prepared the target hosts (ensuring the [relevant ports](#tcp-port-configuration) are accessible) and configured the `inventory_ssh.yaml` file as described above, you're ready to run the playbook.
+
+To check that your inventory file is configured correctly and the control node is able to connect to the target hosts, run:
+
+```bash
+ansible all -m ping -i inventory_ssh.yml
+```
+
+To deploy Community Edition:
+
+```bash
+ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@community-extra-vars.yml"
+```
+
+> **Note:** The playbook takes around 30 minutes to complete.
+
+Once the playbook is complete, Ansible displays a play recap to let you know that everything is done, similar to the following:
+
+```bash
+PLAY RECAP *******************************************************************************************************
+activemq_1                 : ok=24   changed=0    unreachable=0    failed=0    skipped=17   rescued=0    ignored=0
+database_1                 : ok=20   changed=0    unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+nginx_1                    : ok=21   changed=8    unreachable=0    failed=0    skipped=8    rescued=0    ignored=0
+repository_1               : ok=92   changed=43   unreachable=0    failed=0    skipped=14   rescued=0    ignored=0
+search_1                   : ok=34   changed=13   unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+transformers_1             : ok=81   changed=10   unreachable=0    failed=0    skipped=44   rescued=0    ignored=0
+```
+
+Once Community Edition has initialized, access the system using the following URLs with a browser:
+
+* Share: `http://<nginx-host-ip>/share`
+* Repository: `http://<nginx-host-ip>/alfresco`
+* API Explorer: `http://<nginx-host-ip>/api-explorer`
+
+## Useful information
+
+The following section contains further information about the Ansible installation approach.
+
+### Folder structure
+
+A consistent folder structure is used regardless of the role and connection type. You'll find the installed files in the following locations:
+
+| Path | Purpose |
+| ---- | ------- |
+| `/opt/alfresco`     | Binaries |
+| `/etc/opt/alfresco` | Configuration |
+| `/var/opt/alfresco` | Data |
+| `/var/log/alfresco` | Logs |
+
+### Service configuration
+
+The following `systemd` services are installed, which you can use to stop and start each service:
+
+| Service Name | Purpose |
+| ------------ | ------- |
+| `activemq.service` | ActiveMQ Service |
+| `postgresql-<version>.service` | PostgreSQL DB Service (where `<version>` is 11 for ACS 6.2.N and 13 for ACS 7.x) |
```

Comment:
@gavincornwell In this Community page, should we simplify this line to just `postgres-13.service` (to match line 364?

Reply:
Yes please.